Twig/NodeExpressionExtensionReference.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * Copied from twigphp/twig.
+ *
+ * (c) 2009 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JMS\TranslationBundle\Twig;
+
+/**
+ * Represents an extension call node.
+ * Deprecated since twig 1.23 and removed in 2.0.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ */
+class NodeExpressionExtensionReference extends \Twig_Node_Expression
```

Comment:
I cannot see that `NodeExpressionExtensionReference` is being used. Im probably missing something, can you give me a link? I see that we use `Twig_Node_Expression` though.

Generally, we should not use Twig1 classes/interfaces in our Twig2 extractor. The user will never use Twig1 and Twig2 simultaneously. It is okey if some classes are missing for our Twig1 visitor when we use Twig2.